src/infra/main/template.yaml

```diff
@@ -707,7 +707,7 @@ Resources:
         Variables:
           TABLE_NAME:
             Fn::ImportValue: !Sub "${CoreStackName}-AccountStatusTableName"
-          METRIC_SERVICE_NAME: !Sub "${AWS::StackName}-StatusRetriever"
+          METRIC_SERVICE_NAME: StatusRetrieverFunction
```

Comment:
For some reason, the alarms and metrics where not linked, after this change, the alarms started to work, this may need to be updated